src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ResponseOptions/ResponseOptions.utils.tsx

```diff
@@ -39,13 +40,25 @@ export const getResponseItem = ({
     );
   };
 
+  const renderTimePicker = () => (
+    <TimePickerLineChart
+      color={color}
```

Comment:
y can move to commonProps    color={color}
      minDate={minDate}
      maxDate={maxDate}
      answers={answers}
      versions={versions}